dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheServerA/server.xml

```diff
@@ -11,12 +11,33 @@
     
     <httpSession maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="10m"/>
 
-    <httpSessionCache libraryRef="HazelcastLib" writeContents="ALL_SESSION_ATTRIBUTES"/>
+    <httpSessionCache libraryRef="HazelcastLib" writeContents="ALL_SESSION_ATTRIBUTES">
+      <!-- Use app validation token to make sure we don't accidentally join up with other machines running this test at the same time -->
+      <properties hazelcast.application.validation.token="${hzl.cluster.id}"/>
```

Comment:
Are you sure the hazelcast.application.validation.token property can be specified this way?  My understanding is that the only properties accepted were HAZELCAST_CONFIG_LOCATION, HAZELCAST_INSTANCE_NAME, and HAZELCAST_INSTANCE_ITSELF (although I would be happy to be wrong about that...).